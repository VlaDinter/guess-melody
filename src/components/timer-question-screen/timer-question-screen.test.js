import React from 'react';
import renderer from 'react-test-renderer';

import TimerQuestionScreen from './timer-question-screen.jsx';

it(`TimerQuestionScreen is rendered correctly`, () => {
  const tree = renderer.create(<TimerQuestionScreen
    gameTime={0}
    decrementSecond={jest.fn()}
  />).toJSON();

  expect(tree).toMatchSnapshot();
});
