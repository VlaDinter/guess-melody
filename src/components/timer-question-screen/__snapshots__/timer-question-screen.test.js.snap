// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimerQuestionScreen is rendered correctly 1`] = `
<div
  className="timer__value"
  xmlns="http://www.w3.org/1999/xhtml"
>
  <span
    className="timer__mins"
  >
    0
    0
  </span>
  <span
    className="timer__dots"
  >
    :
  </span>
  <span
    className="timer__secs"
  >
    0
    0
  </span>
</div>
`;
